pkg/controllers/uiplugin/config/korrel8r.yaml

```diff
@@ -0,0 +1,20 @@
+# Default configuration for deploying Korrel8r as a service in an OpenShift cluster.
+# Store service URLs assume that stores are installed in their default locations.
+stores:
+  - domain: k8s
+  - domain: alert
+    metrics: https://{{ .Metric }}.openshift-monitoring.svc:9091
```

Comment:
Might be good to make the namespaces variables as well. Even if they're fixed today, they might become variables at some point. Not a reason to delay this PR but on the to-do list

Reply:
Agree, it's an easy thing to do and could be good for the future us.

Reply:
Done